### PR TITLE
INTLY-7813 Add new alerts for high node cpu & memory utilisation

### PR DESCRIPTION
**Pull request by david-martin**
## Additional Information
https://issues.redhat.com/browse/INTLY-7813

## Verification Steps
As the verifier of the PR the following process should be done:

* Do an installation
* Verify the new alerts show in the Prometheus UI
* Run a pod(s) that uses a lot of cpu and/or memory
* Verify the Alerts fire as the node where the pod(s) are running are maxed out on cpu and/or memory

For upgrade:

* Install from master
* The new alerts *shouldn't* exist in Prometheus UI
* Perform an upgrade
* Verify the new alerts show in the Prometheus UI

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

## Checklist

- [x] Updated [Changelog](https://github.com/integr8ly/installation/blob/master/CHANGELOG.md)
- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

Is an upgrade task required and are there additional steps needed to test this?
- [X] Yes
- [ ] No

### CODE REVIEW

File: roles/middleware_monitoring_config/templates/kube_state_metrics_alerts.yml.j2 [hunk 1]
```diff
@@ -172,6 +172,24 @@ spec:
...
           severity: warning
+      - alert: NodeCPUUtilisationHigh
+        annotations:
+          message: The node {{ '{{' }} $node {{ '}}' }}% has been using more than 80% of its CPU for longer than 15 minutes.
+          sop_url: https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts/Cluster_Schedulable_Resources_Low.asciidoc
+        expr: |
+           node:node_cpu_utilisation:avg1m * on(node) group_left() (kube_node_labels{label_node_role_kubernetes_io_compute='true'} == 1) > 0.8
+        for: 15m
+        labels:
+          severity: warning
```
Comment (david-martin) [inline roles/middleware_monitoring_config/templates/kube_state_metrics_alerts.yml.j2:183]: @pat-cremin Can I get your input on severity & thresholds for node cpu & memory utilisation alerts for 1.x?

**Comment by austincunningham** [comment]
## Verified install
image

**Comment by austincunningham** [comment]
## Verifying  cpu load trigger 
with steps outline https://gitlab.cee.redhat.com/integreatly-qe/integreatly-test-cases/-/merge_requests/225/diffs#diff-content-a6cacc73f05da0f8656bc6331fa31cb532168440
image

File: roles/middleware_monitoring_config/templates/kube_state_metrics_alerts.yml.j2 (hunk 1 above)
Comment (david-martin) [reply roles/middleware_monitoring_config/templates/kube_state_metrics_alerts.yml.j2:183]: @Nanyte25 or @alanmoran Can I get your input on severity & thresholds for these new 1.x alerts for Node CPU & Memory utilisation?

**Comment by austincunningham** [comment]
## Verify memory load triggered

unable to verify with https://gitlab.cee.redhat.com/integreatly-qe/integreatly-test-cases/-/merge_requests/225/diffs#diff-content-a6cacc73f05da0f8656bc6331fa31cb532168440 , the pod restarts before getting to 0.8, can verify the statement works as the graph in Prometheus returns the values
image

**Comment by austincunningham** [comment]
install successfull
```bash
TASK [debug] *****************************************************************************************************************
task path: /home/ec2-user/installation/playbooks/install.yml:51
ok: [127.0.0.1] => {
    "msg": "All services have been provisioned successfully."
}

TASK [debug] *****************************************************************************************************************
task path: /home/ec2-user/installation/playbooks/install.yml:53
ok: [127.0.0.1] => {
    "msg": "Please add 'https://launcher-sso-launcher.apps.austin-baf2.open.redhat.com/auth/realms/launcher_realm/broker/github/endpoint' as the Authorization callback URL of your GitHub OAuth Application."
}
META: ran handlers
META: ran handlers

PLAY RECAP *******************************************************************************************************************
127.0.0.1                  : ok=1336 changed=488  unreachable=0    failed=0   
```

**Comment by austincunningham** [comment]
upgrade successful 
```bash
PLAY RECAP *******************************************************************************************************************
127.0.0.1                  : ok=146  changed=59   unreachable=0    failed=0   
```
https://issues.redhat.com/secure/attachment/12475484/integreatly-upgrade.log

**Comment by austincunningham** [comment]
/lgtm
/approve
@jessesarn

**Pull request merged**